stratum/hal/lib/tdi/es2k/es2k_sde_meter.cc

```diff
@@ -109,7 +109,8 @@ ::util::Status Es2kSdeWrapper::WritePktModMeter(
   return ::util::OkStatus();
 }
 
-util::Status GetMeterField(TdiPktModMeterConfig& cfg, std::string field_name,
+util::Status GetMeterField(TdiPktModMeterConfig& cfg,
+                           const std::string field_name,
```

Comment:
```suggestion
const std::string& field_name,
```
`const std::string field_name` makes a copy of the string
`const std::string field_name&` passes a non-null pointer to an existing string

**OPTIMIZATION**